### DISCUSSION TITLE
Restore from a development version database

### DISCUSSION THREAD

**Discussion by oblivcheck** [ppy/osu#30478] [Q&A] [answered]
I build Lazer from source and it auto read my local database, 
now I can't read the database using the Release version:(
![manual_2024-11-02_21-39-12](https://github.com/user-attachments/assets/a419ad81-704d-4622-8dca-86fb5bdcb870)

**Comment by oblivcheck** [comment]
And i can't start multiplayer

**Comment by SupDos** [comment, marked as answer]
You ran it in release mode, which reads from the normal folders. If you run it in debug mode then it uses a separate database (mentioned in https://github.com/ppy/osu/blob/master/CONTRIBUTING.md)

As per https://github.com/ppy/osu/discussions/30371#discussioncomment-11001458, there's nothing that can be done, you'll need to wait until the next version comes out
